### DISCUSSION TITLE
npm install搞不定

### DISCUSSION THREAD

**Discussion by yhtqwe** [Yin-Hongwei/vue-spring-music#76] [Q&A]
终端显示的是npm WARN deprecated babel-eslint@8.2.6: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.
npm WARN deprecated eslint-loader@1.9.0: This loader has been deprecated. Please use eslint-webpack-plugin
等等，该怎么解决？

**Comment by Yin-Hongwei** [comment]
一般是node版本的问题，v12.4.0应该没问题，你换个node版本，node_modules和package.lock.json删了重新执行 npm i
